test/plugin/scenarios/rabbitmq-scenario/config/expectedData.yaml

```diff
@@ -0,0 +1,100 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+registryItems:
+  applications:
+  - {rabbitmq-scenario: nq 0}
+  instances:
+  - {rabbitmq-scenario: 1}
+  operationNames:
+  - rabbitmq-scenario: [/rabbitmq-scenario/case/rabbitmq,/rabbitmq-scenario/case/healthcheck,RabbitMQ/Topic/Queue/test/Producer,RabbitMQ/Topic/Queue/test/Consumer]
+segmentItems:
+- applicationCode: rabbitmq-scenario
+  segmentSize: ge 2
+  segments:
```

Comment:
Should be changed to ```gt 2``` or ```ge 3```

Reply:
@viswaramamoorthy could you follow this?